multiaccounts/settings_social_links/database.go

```diff
@@ -87,3 +115,31 @@ func (s *SocialLinksSettings) SetSocialLinks(links *identity.SocialLinks) (err e
 
 	return nil
 }
+
+func (s *SocialLinksSettings) UpdateSocialLinkFromSync(link *identity.SocialLink) error {
```

Comment:
If possible, could you please cover this with tests in `multiaccounts/settings_social_links/database_test.go`?

Reply:
because there is already a [test](https://github.com/status-im/status-go/blob/e72314c2ec5293c15dfdf594d65881d622ad1352/protocol/messenger_backup_test.go#L143) which will call `UpdateSocialLinkFromSync` internally, so i ignored it, is it okay? @osmaczko